plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixConnectorTest.java

```diff
@@ -796,6 +799,283 @@ public void testExecuteProcedure()
         }
     }
 
+    @Test
+    public void testMergeUpdatePartial()
```

Comment:
This is a very nice test, and just what I requested!

Reply:
Glad to hear! Thanks for the guidance!